### PR TITLE
Document new concurrency policy default - "Queue"

### PR DESCRIPTION
**Pull request by marcinzaremba**
We are changing the default concurrency policy to `Queue` in k6 cloud.

### CODE REVIEW

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md [hunk 1]
```diff
@@ -226,8 +226,8 @@ If you need to increase this limit, contact our support team.
 Additionally, you can change the concurrency limit policy that defines how k6 Cloud acts when the organization reaches the limit and a new test run is triggered.
 Two options are available:
 
-- `Abort test`: the new test run will be automatically aborted. This is the default option.
-- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours.
+- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours. This is the default option.
```
Comment (marcinzaremba) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:229]: Question: we are changing the default policy, but only for newly created organizations (from today onwards). We preserve the previous default for the existing organizations not to confuse them (they might rely on the old default policy). Should we also mention it here in the docs, or is this to technical of a detail?

Comment (marcinzaremba) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:229]: Yes, absolutely. We are just not switching them automatically.

Comment (MattDodsonEnglish) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:229]: I think we should create an admonition in this bullet. Here, I've written it, but do you mind filling in the values:

```
<Blockquote mod="note" title="">

On <DATE>, the default timeout changed from 6 hours to <new_value>.
To avoid confusion, k6 kept the old default for organizations who joined before the change.

If your organization uses the old default and wants to upgrade, change these settings in <location_of_resource>. 

</Blockquote>
```

Comment (ppcano) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:229]: @marcinzaremba correct me if I am wrong, but we only change which status will be the current default policy - the behaviour of both statuses are the same. 

If that is the case, I think we can keep it simple and just add to your changes:  `This is the current default policy`.

Comment (ppcano) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:229]:
```suggestion
- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours. This is the current default option.
```

**Review by ppcano — APPROVED** [review]
LGTM - suggested a minor change as commented (cc @MattDodsonEnglish)

and welcome to the docs Marcin!

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md (hunk 1 above)
Comment (marcinzaremba) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md]: So Pepe's feedback has been included. I think what Matt suggested is also a good addition.
I can imagine a following scenario (not sure how likely it is):
If I were an existing customer just reading the docs, and I were interested in `the concurrency policy`, I could have just assumed, that I do not have to do anything, because the `queue` policy is already set (the default).
I proceed with my testing and my test is aborted. I am surprised. I go to the settings and I see that my policy is `Abort`. Why? I think the docs might be wrong.

Comment (marcinzaremba) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md]: I've added the suggested note. @MattDodsonEnglish Can you please take a look?

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md [hunk 2]
```diff
@@ -226,16 +226,27 @@ If you need to increase this limit, contact our support team.
...
+- `Abort test`: the new test run will be automatically aborted.
+
+<Blockquote mod="note" title="Concurrency limit policy default change">
+
+On 31.01.2023, the default concurrency policy changed from `Abort test` to `Queue test`.
+To avoid confusion, k6 kept the old default for organizations who joined before the change.
+
+If your organization uses the old default and wants to upgrade to `Queue test` policy, simply follow the instructions below.
+
+</Blockquote>
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:239]:
```suggestion
<Blockquote mod="note" title="Changes to the default concurrency limit policy">

On January 1st, 2023, the default concurrency policy changed from `Abort test` to `Queue test`.
To avoid confusion, k6 kept the old default for organizations who joined before the change.

If your organization uses the old default and wants to upgrade to `Queue test` policy,change the policy at **Organization settings > Settings**.

</Blockquote>
```

Comment (MattDodsonEnglish) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:239]: Explanation of my notes:

- It can be hard to parse a huge stack of adjectives, so I add a preposition and removed a word
- Dates formats will confuse different audiences. Better to write it (Or use ISO, at least :-) ).
- It seems like the instructions are just one line, we can be a little redundant. IMO better UX than directing user to a new section.

Comment (MattDodsonEnglish) [reply src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:239]: If these look okay to you, it's fine to me.

**Comment by MattDodsonEnglish** [comment]
If my edits look okay and are committed, this LGTM. Thanks! 

@marcinzaremba do you mind merging when it's ready. I'm not sure when the feature ships. You can find info about deploying:

https://github.com/grafana/k6-docs/tree/main/CONTRIBUTING

File: src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md [hunk 3]
```diff
@@ -226,16 +226,27 @@ If you need to increase this limit, contact our support team.
...
-- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours.
+- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours. This is the current default option.
+- `Abort test`: the new test run will be automatically aborted.
+
+<Blockquote mod="note" title="Changes to the default concurrency policy">
+
+On January 31st, 2023, the default concurrency policy changed from `Abort test` to `Queue test`.
+To avoid confusion, k6 kept the old default for organizations who joined before the change.
+
+If your organization uses the old default and wants to upgrade to `Queue test` policy, change the policy at **Organization settings > Settings**.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md:237]:
```suggestion
If your organization uses the old default and wants to upgrade to `Queue test` policy, change the policy as described in the following section.
```

**Comment by MattDodsonEnglish** [comment]
> It seems like the instructions are just one line, we can be a little redundant. IMO better UX than directing user to a new section.

Sorry, I wasn't looking very hard, I didn't realize that section was one line and immediately below the admonition. Fixed with [0dc4426](https://github.com/grafana/k6-docs/pull/1015/commits/0dc4426fe201db33362b3d90f29a12c0a378effc)

**Review by MattDodsonEnglish — APPROVED** [review]
